Extensions/PathfindingBehavior/tests/commonpathfindingruntimebehavior.spec.js

```diff
@@ -1,5 +1,8 @@
-var commonpathfindingtest = {
-  createScene: () => {
+// tests cases where every collisionMethod has the same behavior.
```

Comment:
Do you think it would work to move these changes to the tests in a separate PR?
(apart from `'Legacy', 'AABB', 'HitBoxes'` and a few collision method related stuff).

The idea would be to make these changes in the tests of this PR as small as possible.